src/main/java/org/jabref/gui/maintable/BibEntryTableViewModel.java

```diff
@@ -116,10 +119,13 @@ public BibEntry getEntry() {
         ArrayList<Observable> observables = new ArrayList<>(List.of(entry.getObservables()));
         observables.add(fieldValueFormatter);
 
-        value = Bindings.createStringBinding(() ->
-                        fieldValueFormatter.getValue().formatFieldsValues(fields, entry),
-                observables.toArray(Observable[]::new));
+        value = Bindings.createStringBinding(() -> fieldValueFormatter.getValue().formatFieldsValues(fields, entry),
+                                             observables.toArray(Observable[]::new));
         fieldValues.put(fields, value);
         return value;
     }
+
+    public ObservableValue<String> getBibDatabaseContextPath() {
+        return new ReadOnlyStringWrapper(bibDatabaseContext.getDatabasePath().map(Path::toString).orElse(""));
+    }
```

Comment:
Not sure if this is a good idea, to make this an ObservableValue, which implies that it can be changed from the outside view on the signature of the method. Convention was in other cases to use Properties (in this case ReadOnlyStringProperty)

Reply:
It's also only a static string, or I'm missing something?

Reply:
Yes, it's used in the Cell(Value)Factory for the column and therefore needs to be wrapped in a property 
I will fix the codestyle formatting later.

Reply:
Then just use https://github.com/JabRef/jabref/blob/bbb39a8ffb3b866c2081abfbd43dfce1da73ac1f/src/main/java/org/jabref/gui/util/BindingsHelper.java#L135 in the cell factory.

Reply:
I disagree.
It's not a static string if the name of the library can change (e.g. save as) the string changes. Convention was until now to provide those variables as properties, not as observables.

Reply:
I added this now as a property